modules/prebidServerBidAdapter/index.js

```diff
@@ -832,7 +832,8 @@ const OPEN_RTB_PROTOCOL = {
           bidObject.currency = (response.cur) ? response.cur : DEFAULT_S2S_CURRENCY;
 
           // TODO: Remove when prebid-server returns ttl and netRevenue
-          bidObject.ttl = (bid.ttl) ? bid.ttl : DEFAULT_S2S_TTL;
+          const configTtl = _s2sConfig.defaultS2sTtl || DEFAULT_S2S_TTL;
+          bidObject.ttl = (bid.ttl) ? bid.ttl : configTtl;
```

Comment:
the bid object coming from Prebid Server can't put ttl directly in the seatbid[].bid[] object because 'ttl' isn't a standard openrtb field. Please look for it in bidObject.ext.ttl.